.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
add `dist/bundle.js` to gitignore. Generated files should not be in sourse control